app/src/main/java/jp/developer/retia/frozenword/ui/watch/WatchScreen.kt

```diff
@@ -111,13 +133,85 @@ fun EditMemoScreen(
     }
 }
 
+@Composable
+fun LogsScreen(modifier: Modifier = Modifier, logs: List<Log>) {
+    LazyColumn {
+        itemsIndexed(logs) { i, item ->
+            val before = logs.getOrNull(i - 1)
+            val next = logs.getOrNull(i + 1)
+            LogCard(
+                iconTitle = "${item.time.month + 1}/${item.time.date}",
+                message = item.message,
+                date = item.time.toString(),
+                showBottomLine = next != null && item.time.date + 1 == next.time.date,
+                showTopLine = before != null && item.time.date - 1 == before.time.date
+            )
+        }
+
+    }
+}
+
+@Preview
+@Composable
+fun PreviewLogCard() {
+    FrozenWordTheme {
+        Surface {
+            LogCard(iconTitle = "1/16", message = "腕立て10回", date = "1/16 20:00")
+        }
+    }
+}
+
+@Composable
+fun LogCard(
+    modifier: Modifier = Modifier,
+    iconTitle: String,
+    message: String,
+    date: String,
+    showBottomLine: Boolean = false,
+    showTopLine: Boolean = false
+) {
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .wrapContentHeight()
+            .padding(start = 8.dp, end = 8.dp)
+    ) {
+        Column(horizontalAlignment = Alignment.CenterHorizontally) {
+            val endLineWidth = if (showTopLine) 2 else 0
+            Divider(
+                color = Color.Red, modifier = Modifier
```

Comment:
```suggestion
                color = Color.Red,
                modifier = Modifier
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>